frontend/components/site/ScanConfig/index.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import api from '../../../util/federalistApi';
+import { BUILD_SCAN_RULES } from '../../../util/scanConfig';
+import { useDefaultScanRules } from '../../../hooks/useDefaultScanRules';
+import { useSiteBuildTasks } from '../../../hooks/useSiteBuildTasks';
+
+function appMatch(type) {
+  // TODO: somehow remove this function or unduplicate from api utils
+  // get a readable string from a BuildTaskType appName to match the scan rule types
+  return type.metadata.appName.replace(/pages-(.*)-task-.*/, '$1');
+}
+
+function configToRuleArray(config) {
```

Comment:
Should this be in the useDefaultScansHook so we get the data structure we expect via the hook?